source/analysis/AM1DCalibrationAB.h

```diff
@@ -40,7 +40,7 @@ class AM1DCalibrationAB : public AMStandardAnalysisBlock
 	Q_OBJECT
 
 	Q_PROPERTY(QString dataName READ dataName WRITE setDataName)
-	Q_PROPERTY(QString NormalizationName READ NormalizationName WRITE setNormalizationName)
+	Q_PROPERTY(QString NormalizationName READ normalizationName WRITE setNormalizationName)
```

Comment:
I like that these properties are now the proper camel-case. Did this change solve a particular problem (I'm thinking of #2256)?